### ISSUE TITLE
Issues with React 0.14

### ISSUE THREAD

**Issue by yashprit** [tomchentw/react-toastr#35]
Looks like still some work is require to make it compatible with react 0.14

```Unhandled promise rejection Error: Invariant Violation: Objects are not valid as a React child (found: Error: Unauthorized). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons. Check the render method of `ToastMessage.animation`.```

To Learn about error https://github.com/facebook/react/issues/5139

Would accept pull request for above issue?

**Comment by tomchentw** [comment]
Sure welcome for the PR. Where did you find this error BTW?

**Comment by polkovnikov-ph** [comment]
Not only that, BTW. `var React = require("react/addons");` is deprecated.
```
Warning: require('react/addons') is deprecated. Access using require('react-addons-{addon}') instead.
```

**Comment by tineo** [comment]
I got this.
```
Warning: ToastMessage.animation.getDOMNode(...) is deprecated. Please use  ReactDOM.findDOMNode(instance) instead
```

**Comment by jerairrest** [comment]
I've put in a PR to hopefully fix all of these warnings. :D

**Comment by polkovnikov-ph** [comment]
:+1:

I'll test it too as soon as I'll get some free time.

@tomchentw

**Comment by tomchentw** [comment]
Closed by #38. Released [v2.2.0](https://github.com/tomchentw/react-toastr/releases/tag/v2.2.0)
